### ISSUE TITLE
Inconsistent alignment of 'attachment' icon in comments page and some changes in voting part.

### ISSUE THREAD

**Issue by dikshashukla455** [EOS-uiux-Solutions/user-story#214]
## Describe the bug

There are two bugs on the Comments page:
1) There is an inconsistent alignment of the attachment icon in an input box of the comment.
2) In a number of votes if there is only 1 vote then it should write 1 'vote' instead of 'votes' and for 0 and numbers other than 1 it should write 'votes'.

## Actual Behavior

![Screenshot (1009)_LI](https://user-images.githubusercontent.com/60606998/161137355-08ecdfb6-7894-4c74-a71a-993ae59c4006.jpg)

You can see the bugs in the above image, which I have underlined.

## To Reproduce

1. Go to someone's comments page by clicking on that story that is present on the home page.

## Expected behavior

![Screenshot (1006)](https://user-images.githubusercontent.com/60606998/161138987-2f1e6e86-7677-4603-8a73-8dc37b98b1c5.png)

By resolving all the bugs, The page will look like same as the above image.

## Environment (please complete the following information):

- OS: Windows 10
- Browser: Chrome

**Comment by dikshashukla455** [comment]
@cyntss @mharshita @SundeepChand I would like to work on this issue and fix it.

**Comment by sarthakkundra** [comment]
Good catch @dikshashukla455! Great to see your first official issue opened at EOS. Let me know if you need any help with this :)

**Comment by dikshashukla455** [comment]
> Good catch @dikshashukla455! Great to see your first official issue opened at EOS. Let me know if you need any help with this :)

Yeah, sure! Thank you 😊.
